pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -216,6 +218,17 @@ func UseKeyringRingSvc() bool {
 	handleError(err)
 	return optionImageRegistry == "yes"
 }
+func SelectRoute() string {
+	var optionRoute string
+	prompt := &survey.Input{
+		Message: "Provide a route name for your application ?",
+		Help:    "If you specify the route flag and pass the string, that string will be in the route.yaml that is generated",
```

Comment:
Although technically should be part of the bootstrap PR, could you also add help for the target port?